static/library/utils.ts

```diff
@@ -41,6 +41,17 @@ export function convertArrayAttribute(attributeValue: string): string[] {
   }
 }
 
+/**
+ * Custom attribute converter for boolean type attributes.
+ * Checks that the attribute value equals "true"
+ *
+ * @param attributeValue the attribute value provided to the web component
+ * @returns boolean
+ */
+export function convertBooleanAttribute(attributeValue: string): boolean {
+  return attributeValue.toLowerCase() !== "false";
```

Comment:
might be better to explicitly test for "true"?

Reply:
that's what I originally wanted to do, but therea re example cases like these https://github.com/datacommonsorg/website/blob/45a226a04d47d0296849afe61c3ce3fe87376fa3/packages/web-components/examples/all-charts.html#L158-L159 that need to evaluate to true ..

Reply:
Ah - got it. A comment to that effect would be good to have.

Reply:
udpated in https://github.com/datacommonsorg/website/pull/3595